src/app/modules/shared/components/details-fields/details-fields.component.spec.ts

```diff
@@ -314,6 +319,19 @@ describe('DetailsFieldsComponent', () => {
     expect(component.saveEntry.emit).toHaveBeenCalledWith(data);
   });
 
+  it('onSubmit an entry without change hours, should not modify the start_date and end_date ', () => {
+    component.entryToEdit = {...entryToEdit, description: 'test', };
+    fixture.componentInstance.ngOnChanges();
+
+    const startHourValue = moment().subtract(3, 'hours').format('HH:mm');
+    const endHourValue = moment().format('HH:mm');
+
+    component.onSubmit();
+
+    expect(component.starDateValue).toEqual(startHourValue);
+    expect(component.endDateValue).toEqual(endHourValue);
```

Comment:
How are you asserting that the other fields, like description are being updated?

Reply:
there's a new test to prove that.